File: process-management-api/src/main/java/com/victorsantos/processmanagementapi/users/usercases/admin/getuser/GetUserUserCaseResponse.java
package com.victorsantos.processmanagementapi.users.usercases.admin.getuser;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class GetUserUserCaseResponse {
  private String id;
  private String name;
  private String email;
  private String role;
}

File: process-management-web/src/users/admin/usercases/get-user/GetUserUserCaseResponse.ts
export default class GetUserUserCaseResponse {
  id: string;
  name: string;
  role: string;
  email: string;

  constructor(attrs: {
    id: string;
    name: string;
    role: string;
    email: string;
  }) {
    this.id = attrs.id;
    this.name = attrs.name;
    this.role = attrs.role;
    this.email = attrs.email;
  }
}
